.github/workflows/build.yaml

```diff
@@ -30,10 +30,22 @@ jobs:
       uses: actions/setup-python@d09bd5e6005b175076f227b13d9730d56e9dcfcb
       with:
         python-version: ${{ matrix.python }}
-    - name: Install dependencies
-      run: make setup
+    - name: Create and activate virtual environment
+      if: ${{ matrix.os == 'ubuntu-latest' || matrix.os == 'macos-latest' }}
```

Comment:
Shouldn't this if statement be:
```
${{ matrix.os }} == 'ubuntu-latest' || ${{ matrix.os }} == 'macos-latest' 
```

Reply:
Not according to original conversation on the Action: https://github.com/actions/setup-python/issues/359#issuecomment-1080679055

Also, the tests run successfully, so guess that code works 🤓

Reply:
You don't need to evaluate the [expression](https://docs.github.com/en/actions/learn-github-actions/expressions#about-expressions) in `if` statements.

> When you use expressions in an if conditional, you may omit the expression syntax (${{ }}) because GitHub automatically evaluates the if conditional as an expression. 

See this [example](https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#example-only-run-job-for-specific-repository).

Reply:
It’s about consistency: we use `${{ … }}` in all workflows, so I wouldn’t want to change that here in this one instance just because, why?